src/main/java/com/google/cloud/tools/appengine/operations/DevServerV1.java

```diff
@@ -272,29 +272,14 @@ boolean isSandboxEnforced(List<Path> services) throws AppEngineException {
   }
 
   /**
-   * Gets the App Engine runtime ID for Java runtimes. Order of precedence is 11 > 8 > 7. With
-   * default being 7.
+   * Gets the App Engine runtime ID for Java runtimes.
+   *
+   * @param isSandboxEnforced if {@code true}, use Java 8; otherwise, use Java 7
+   * @return "java8" if {@code isJava8} is true; otherwise, returns "java7"
    */
   @VisibleForTesting
-  static String getGaeRuntimeJava(List<Path> services) throws AppEngineException {
-    boolean hasJava8 = false;
-    for (Path serviceDirectory : services) {
-      Path appengineWebXml = serviceDirectory.resolve("WEB-INF/appengine-web.xml");
-      try (InputStream is = Files.newInputStream(appengineWebXml)) {
-        String serviceRuntime = AppEngineDescriptor.parse(is).getRuntime();
-        if (serviceRuntime.startsWith("java11")) {
-          return "java11";
-        } else if (serviceRuntime.startsWith("java8")) {
-          hasJava8 = true;
-        }
-      } catch (IOException | SAXException ex) {
-        throw new AppEngineException(ex);
-      }
-    }
-    if (hasJava8) {
-      return "java8";
-    }
-    return "java7";
+  static String getGaeRuntimeJava(boolean isSandboxEnforced) {
```

Comment:
The argument seems incorrect, or confusing at least, because you pass the argument after negating it: `getGaeRuntimeJava(!isSandboxEnforced)`. Indeed, the sandbox is _not_ enforced for Java 8 runtimes. Chould be `noSandbox` or `sandboxNotEnforced`, but I don't like them either honestly.

Reply:
Hrmmm... yeah I was reverting to previous state. However, I might as well clean this up.

Reply:
@chanseokoh I need to file a followup that removes java7 support anyway. I'd like to leave this PR as a simple revert if that's okay.

Reply:
Yeah, sounds good.